roles/neutron-common/defaults/main.yml

```diff
@@ -79,6 +80,36 @@ neutron:
     service_plugin:
       - neutron_lbaas.services.loadbalancer.plugin.LoadBalancerPluginv2
     service_provider: LOADBALANCERV2:Haproxy:neutron_lbaas.drivers.haproxy.plugin_driver.HaproxyOnHostPluginDriver:default
+    f5:
+      enabled: False
+      debug: False
+      periodic_interval: 10
+      f5_ha_type: standalone
+      f5_external_physical_mappings: "default:1.2:True"
+      f5_vtep_folder: 'Common'
+      f5_vtep_selfip_name: 'vtep'
+      advertised_tunnel_types: vxlan
+      f5_populate_static_arp: False
+      l2_population: True
+      f5_global_routed_mode: False
+      use_namespaces: True
+      max_namespaces_per_tenant: 1
+      f5_route_domain_strictness: False
+      f5_snat_mode: True
+      f5_snat_addresses_per_subnet: 1
+      f5_common_external_networks: False
+      f5_bigip_lbaas_device_driver: f5_openstack_agent.lbaasv2.drivers.bigip.icontrol_driver.iControlDriver
+      icontrol_hostname: 256.256.256.256
+      icontrol_username: admin
+      icontrol_password: admin
+      f5_parent_ssl_profile: clientssl
+      service_provider: LOADBALANCERV2:F5Networks:neutron_lbaas.drivers.f5.driver_v2.F5LBaaSV2Driver:default
+      f5_service_provider: "https://github.com/F5Networks/neutron-lbaas/releases/download/v9.1.0/f5.tgz"
```

Comment:
The f5 related code is out of date in openstack/neutron-lbaas, and we have to override neutron_lbaas/drivers/f5/driver_v2.py by downloading f5.tgz as per f5 documentation guide. http://f5-openstack-lbaasv2-driver.readthedocs.io/en/latest/map_quick-start-guide.html